cypress/support/commands.js

```diff
@@ -231,7 +232,7 @@ Cypress.Commands.add('handleDatePicker', (startDatePicker, endDatePicker) => {
         day: '2-digit',
     };
     const _date = new Date();
-    _date.setDate(_date.getDate() + 2);
+    _date.setDate(_date.getDate() + 4);
```

Comment:
doesn't have to stay at an additional 4 days. Just wanted to make sure it would test past the end of this month correctly.